packages/dm-core/src/hooks/useList.tsx

```diff
@@ -52,115 +67,152 @@ export function useList<T>(idReference: string): IUseListReturnType<T> {
       .then((response: AxiosResponse) => {
         setAttribute(response.data)
       })
-      .catch((error) => toast.error(error))
+      .catch((error: AxiosError<ErrorResponse>) => {
+        setError(error.response?.data || { message: error.name, data: error })
+      })
       .finally(() => setLoading(false))
   }, [dmssAPI, idReference])
 
   useEffect(() => {
     if (!attribute) return
     setLoading(true)
-    dmssAPI
-      .documentGet({
-        address: idReference,
-        depth: 0,
-      })
-      .then(async (response: AxiosResponse) => {
-        if (attribute.contained) {
-          const items = Object.values(response.data).map((data, index) => ({
-            key: crypto.randomUUID(),
-            index: index,
-            data: data,
-            reference: null,
-          }))
-          // @ts-ignore
-          setItems(items)
-          setError(null)
-        } else {
-          const resolved = await dmssAPI.documentGet({
-            address: idReference,
-            depth: 1,
-          })
-          if (Array.isArray(resolved.data)) {
+
+    const effect = async () => {
+      setDirtyState(false)
+      dmssAPI
+        .documentGet({
+          address: idReference,
+          depth: 0,
+        })
+        .then(async (response: AxiosResponse) => {
+          if (attribute.contained) {
+            if (!Array.isArray(response.data)) {
+              throw new Error(
+                `Not an array! Got document ${JSON.stringify(response.data)} `
+              )
+            }
+
             const items = Object.values(response.data).map((data, index) => ({
-              key: crypto.randomUUID(),
+              key: uuidv4(),
```

Comment:
Why use extra package for uuid when it's supported in stdlib?

Reply:
Done 👍 Let's just use the default crypto instead.